ansible/roles/real-postfix/templates/postfix/virtual-alias-maps-relay-ldap.cf.j2

```diff
@@ -0,0 +1,11 @@
+# Check if the vhost is served from another pool; if so, we relay to
+# that pool's IP. This is also used as a relay_domains map to tell
```

Comment:
I don't see the config to use it as a relay_domains map?

Reply:
Interesting question. This requires further investigation.

Reply:
It should have been and I've added it, after more testing.